measure/build.gradle.kts

```diff
@@ -30,47 +32,58 @@ dependencies {
     // Use the Kotlin JDK 8 standard library.
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 
-    testImplementation("io.kotest:kotest-runner-junit5:4.3.2")
-    testImplementation("io.kotest:kotest-assertions-core:4.3.2")
-    testImplementation("io.kotest:kotest-property:4.3.2")
+    testImplementation("io.kotest:kotest-runner-junit5:5.9.1")
+    testImplementation("io.kotest:kotest-assertions-core:5.9.1")
+    testImplementation("io.kotest:kotest-property:5.9.1")
 }
 
-java {
-    toolchain {
-        languageVersion.set(JavaLanguageVersion.of("17"))
-    }
+kotlin {
+    jvmToolchain(17)
 }
 
-tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
-    kotlinOptions {
-        jvmTarget = "17"
-    }
+java {
+    // We add both the sources and the javadoc jar.
```

Comment:
The comment suggest that we also publish the javadoc. Should we include `withJavaDocJar` or the Kotlin equivalent?

Reply:
We produce a dokka jar instead. I'll change the comment.